library/src/scala/quoted/Quotes.scala

```diff
@@ -2087,6 +2087,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
 
     /** Methods of the module object `val Unapply` */
     trait UnapplyModule { this: Unapply.type =>
+      def apply(fun: Term, implicits: List[Term], patterns: List[Tree]): Unapply
```

Comment:
This needs some documentation explaining what all these parameters correspond to.

Reply:
Added